### DISCUSSION TITLE
Multiple sessions available at xRDP login prompt. How to auto select desired session and continue with login without seeing xRDP login screen?.

### DISCUSSION THREAD

**Discussion by d3al** [neutrinolabs/xrdp#2740] [Q&A]
Upon connecting to xRDP I get the xRDP login prompt which has two sessions options in the dropdown. One of them requires authentication, username and password. The other session does not require credentials and just has the Ok button to connect into the console which is working.

I want to pass in what session to select and if possible auto confirm so I don't see the xrdp login prompt when connecting to the console. Is this something I can configure in connection rdp file or do I need to setup the xRDP config file for the console session console to run on a different port?

**Comment by matt335672** [comment]
What version of xrdp is this, @d3al ? Nothing's documented, but there's some code in there which might meet your use case.
